content/en/docs/reference/config-api/kube-scheduler-config.v1.md

```diff
@@ -0,0 +1,1384 @@
+---
+title: kube-scheduler Configuration (v1)
```

Comment:
Copy of `kube-scheduler-config.v1beta3.md`, just modify the versions.

Reply:
Don't do this. This file is auto-generated.

Reply:
Removed.

Reply:
I have no idea how the non-existent reference `/docs/reference/config-api/kube-scheduler-config.v1` will be auto-generated. But I used the reference in my commit.

Reply:
It's generated through the process at: https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-api/#generating-the-api-reference-docs

I don't rightly know who does it though (i think it's a step in the release process)

Reply:
the scheduler config v1 is not a thing in 'k8s.io/kube-scheduler v0.24.0'. We update the `go.mod` every time a new release is out (i.e. '0.25.0' tag is available in this case), and then regenerate the API reference.

It may be just fine because the updated pages here will not be published before 1.25 is out. When 1.25 is out, we will generate the reference in a day or two.